packages/ts/generator-cli/test/fixtures/version.snap

```diff
@@ -0,0 +1 @@
+24.7.0-alpha11
```

Comment:
Is this something we will have to update manually?

Reply:
If so, would it be better to remove this and check that "should print version" just outputs something that seems a version number? Like, it contains a number and a dot.

Reply:
Ah, sorry, I already faced this issue and just didn't remove the file. Currently, it reads `package.json` and expects the correct version there (yeah, a bit circly, but I guess it's ok for testing a code).